pkg/storage/chunk/cache/fifo_cache.go

```diff
@@ -86,14 +86,21 @@ type FifoCache struct {
 
 	entriesAdded    prometheus.Counter
 	entriesAddedNew prometheus.Counter
-	entriesEvicted  prometheus.Counter
+	entriesEvicted  *prometheus.CounterVec
 	entriesCurrent  prometheus.Gauge
 	totalGets       prometheus.Counter
 	totalMisses     prometheus.Counter
 	staleGets       prometheus.Counter
 	memoryBytes     prometheus.Gauge
 }
 
+const (
+	expiredReason string = "expired" //nolint:staticcheck
+	fullReason           = "full"
+	tooBigReason         = "object too big"
+	stoppedReason        = "stopped"
```

Comment:
`stopped` doesn't give the operator much context, especially the ones who do not go and check the code. I think we should not increment the metric when we are evicting items while stopping. It doesn't make a lot of sense to me.

Reply:
I thought the same actually, cool. Will remove :+1: